### ISSUE TITLE
Page scroll freeze after 'create folder' dialog

### ISSUE THREAD

**Issue by DenisVuyka** [Alfresco/alfresco-ng2-components#1409]
**Type of issue:**  (check with "[x]")
```
- [ ] New feature request
- [x] Bug  
- [ ] Support request
```
**Current behavior:**

After showing 'create folder' dialog the entire page scrolling freezes. It is no longer possible to scroll the page and document list.

**Expected behavior:**

Page should be scrolling

**Steps to reproduce the issue:**

**Component name and version:** 

**Browser and version:** 

 
**Node version (for build issues):**  

**New feature request:**

**Comment by eromano** [comment]
I am not able to reproduce the bug

**Comment by DenisVuyka** [comment]
@eromano try to shrink the browser height a bit to make a page with document list scrollable. As we have removed debug buttons at the bottom it is now a bit harder to reproduce.

**Comment by hashraf88** [comment]
@DenisVuyka I could not reproduce after,
a) shrinking page 
b) entering folder name and clicking create
c) clicking cancel on dialog

Tested on Chrome, Firefox and Safari

**Comment by hashraf88** [comment]
Able to reproduce, issue is only in Chrome

1. Ensure page is scrollable i.e. files/folder added to ensure page is long than one page length
2. Click create > new folder > cancel
3. DO NOT select/highlight any file or folder and try to scroll

**Expected results**
Page is scrollable

**Actual results**
Page is only scrollable once an action occurs e.g. clicking/higlighting a file or folder

[doc list scrolling.mp4.zip](https://github.com/Alfresco/alfresco-ng2-components/files/701608/doc.list.scrolling.mp4.zip)

**Comment by DenisVuyka** [comment]
Will be fixed as part of the #1514
